docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,109 @@
-Assessment
-==========
+Introduction
+============
 
-Metrics
--------
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
 
-.. currentmodule:: fairlearn.metrics
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+1. Identify harms
+-----------------
+
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
+
+2. Identify the groups that might be harmed
+-------------------------------------------
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular application. For example, for 
+speech-to-text transcription, groups based on the regional dialect or being a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+3. Quantify harms
+-----------------
+
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are qualified). One possible metric is the false negative rate: fraction of qualified candidates that are screened out.
+
+* In speech-to-text scenario, the harm could be measured by word error rate, number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, harms might not be readily identifiable. If you 
```

Comment:
Is this issue you're raising here about how to _identify_ harms? Or whether some harms can be _quantified_? I ask because for representational harms, for instance, it is much easier to identify those harms (e.g., with stereotypes in language generation models) than it is to quantify those, which is an open area of research

Reply:
Maybe both? It might be difficult to identify which harms will appear downstream long after a system is implemented (like indirect effects). But I think the intent of this sentence is to discuss harms that are difficult to quantify, so I have changed the sentence accordingly.